frontend/src/metabase/visualizations/lib/LineAreaBarPostRender.js

```diff
@@ -245,6 +245,49 @@ function onRenderVoronoiHover(chart) {
     .order();
 }
 
+// NOTE- this is a total hack just to get things rolling, not sure if this is the right place
+// to do this or not
+function onRenderValueLabels(chart) {
+  // Do nothing if there is nothing to do
+  if (!chart.settings["graph.show_values"]) {
+    return false;
+  }
+
+  // use the chart body so things line up properly
+  const parent = chart.svg().select(".chart-body");
```

Comment:
We seem to layer other text like goal and stuff outside the chart body but I wasn't immediately clear on how to get things to align properly without using the chart body as a starting grid.